### PR TITLE
feature(ui-ux): status api integration

### PR DESCRIPTION
**Pull request by chloezxyy**
#### What this PR does / why we need it:
- Integrate the endpoint `/bridge/status` to check whether if the bridge is up or not

#### Which issue(s) does this PR fixes?:

Fixes #

#### Additional comments?:
- Additional usage of `ApiProvider` wrapper in `Base.tsx` to get `/bridge/status` endpoint to check for condition whether`isUp` is returning true (Bridge is up, don't display maintenance screen)

#### Developer Checklist:

- [x] Read your code changes at least once
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode\*
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [x] Added e2e tests\*
- [ ] Added translations\*

### CODE REVIEW

File: apps/web/src/store/website.tsx [hunk 1]
```diff
@@ -72,13 +72,35 @@ export const bridgeApi = createApi({
   }),
 });
 
+export const statusWebsiteSlice = createApi({
```
Comment (thedoublejay) [inline apps/web/src/store/website.tsx:75]: don't add it here, need to create a new flag for it. Maybe change the name to `status.tsx`

**Review by thedoublejay — CHANGES_REQUESTED** [review]

**Review by thedoublejay — CHANGES_REQUESTED** [review]
Also, can you add a simple e2e on it?

1. Intercept the result and change it to true - check if the maintenance text is displayed
2. If not, check if the actual page is displayed

File: apps/web/src/store/website.tsx (hunk 1 above)
Comment (chloezxyy) [reply apps/web/src/store/website.tsx]: Refactored a lil

**Comment by chloezxyy** [comment]
> Also, can you add a simple e2e on it?
> 
> 1. Intercept the result and change it to true - check if the maintenance text is displayed
> 2. If not, check if the actual page is displayed

Added!

**Pull request merged**